dev/README

```diff
@@ -2,6 +2,38 @@ This directory contains information and tools to help develop the
                              Coq system
                        ======================
 
+Beginners guide
+---------------
+
+The general workflow is to first setup a development environment with
+the correct `configure` settings, then hacking on Coq, make-ing, and testing.
+
+First of all, it is recommended to run the configure with
+    ./configure -local -devel
+
+-local allows `make install` to create a local dev install.
```

Comment:
This is incorrect, rather `-local` means there is no install step, Coq is used from the build directory.